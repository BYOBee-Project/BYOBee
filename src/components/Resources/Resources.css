.resources {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 60%;
}

.resource-images {
  display: flex;
  flex-direction: row;
  justify-content: center;
}

.resources-image {
  width: 15vw;
  margin: 1vw;
}
