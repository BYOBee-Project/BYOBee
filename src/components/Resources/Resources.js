import React from 'react';
import './Resources.css';
import { Link } from 'react-router-dom';
import honeybee from '../../assets/honeybee.jpg';
import bumblebee from '../../assets/bumblebee.jpeg';
import leafcutterbee from '../../assets/leafcutterbee.jpeg';

export default function Resources() {
  return (
    <div className="resources">
      <h2 className="resources-title">Support Our Pollinators!</h2>
      <p>
        There is an incredible diversity of bee species, with ~20,000 in the world, ~4,000 in North
        America and ~500 in Oregon alone! They come in all shapes and sizes, make their homes in a
        variety of unique places and together play an integral role in the pollination of our lands.
      </p>
      <p>
        Unfortunately, due to factors such as climate change, habitat loss, rampant pesticide usage,
        human management practices, and the spread of pests and disease - our bee populations are in
        decline. Here are a few simple ways to support pollinator populations of all kinds. üêùüíõ‚ú®
      </p>
      <div className="resources-images">
        <img className="resources-image" src={honeybee} alt="honeybee" />
        <img className="resources-image" src={bumblebee} alt="bumblebee" />
        <img className="resources-image" src={leafcutterbee} alt="leafcutter bee" />
      </div>
      <ul>
        <li>
          Plant native flowers that bloom at different stages of the season (and make sure they
          haven not been treated with pesticides).
        </li>
        <li>
          Provide habitat for tunnel-nesting bees, and leave brambles and bare, sunny patches in
          your yard for ground-nesters
        </li>
        <li>
          Purchase local and organic produce to increase the demand for bee-friendly farming
          practices.
        </li>
        <li>
          Go pesticide-free! Pest-control products do not discriminate, and will kill bees alongside
          the target pest.
        </li>
        <li>
          Get involved in local citizen science efforts to help entomologist track important data on
          bee populations! Check out the{' '}
          <a href="https://www.oregonbeeproject.org/">Oregon Bee Project</a>.
        </li>
        <li>Spread the word and share your love of bees with others!</li>
      </ul>
      <button>
        <Link to={'/'}>Back to Home</Link>
      </button>
    </div>
  );
}
