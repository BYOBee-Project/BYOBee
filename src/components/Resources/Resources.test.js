import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Resources from './Resources.js';

test('Resource should render our resources information', async () => {
  const container = render(
    <MemoryRouter>
      <Resources />
    </MemoryRouter>
  );
  screen.findByText('Support Our Pollinators!');
  expect(container).toMatchSnapshot();
});
