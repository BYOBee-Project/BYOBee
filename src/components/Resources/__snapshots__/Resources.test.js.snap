// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resource should render our resources information 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="resources"
      >
        <h2
          class="resources-title"
        >
          Support Our Pollinators!
        </h2>
        <p>
          There is an incredible diversity of bee species, with ~20,000 in the world, ~4,000 in North America and ~500 in Oregon alone! They come in all shapes and sizes, make their homes in a variety of unique places and together play an integral role in the pollination of our lands.
        </p>
        <p>
          Unfortunately, due to factors such as climate change, habitat loss, rampant pesticide usage, human management practices, and the spread of pests and disease - our bee populations are in decline. Here are a few simple ways to support pollinator populations of all kinds. üêùüíõ‚ú®
        </p>
        <div
          class="resources-images"
        >
          <img
            alt="honeybee"
            class="resources-image"
            src="honeybee.jpg"
          />
          <img
            alt="bumblebee"
            class="resources-image"
            src="bumblebee.jpeg"
          />
          <img
            alt="leafcutter bee"
            class="resources-image"
            src="leafcutterbee.jpeg"
          />
        </div>
        <ul>
          <li>
            Plant native flowers that bloom at different stages of the season (and make sure they haven not been treated with pesticides).
          </li>
          <li>
            Provide habitat for tunnel-nesting bees, and leave brambles and bare, sunny patches in your yard for ground-nesters
          </li>
          <li>
            Purchase local and organic produce to increase the demand for bee-friendly farming practices.
          </li>
          <li>
            Go pesticide-free! Pest-control products do not discriminate, and will kill bees alongside the target pest.
          </li>
          <li>
            Get involved in local citizen science efforts to help entomologist track important data on bee populations! Check out the
             
            <a
              href="https://www.oregonbeeproject.org/"
            >
              Oregon Bee Project
            </a>
            .
          </li>
          <li>
            Spread the word and share your love of bees with others!
          </li>
        </ul>
        <button>
          <a
            href="/"
          >
            Back to Home
          </a>
        </button>
        <p
          class="credit"
        >
          All photos attributed to Emily Parker.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="resources"
    >
      <h2
        class="resources-title"
      >
        Support Our Pollinators!
      </h2>
      <p>
        There is an incredible diversity of bee species, with ~20,000 in the world, ~4,000 in North America and ~500 in Oregon alone! They come in all shapes and sizes, make their homes in a variety of unique places and together play an integral role in the pollination of our lands.
      </p>
      <p>
        Unfortunately, due to factors such as climate change, habitat loss, rampant pesticide usage, human management practices, and the spread of pests and disease - our bee populations are in decline. Here are a few simple ways to support pollinator populations of all kinds. üêùüíõ‚ú®
      </p>
      <div
        class="resources-images"
      >
        <img
          alt="honeybee"
          class="resources-image"
          src="honeybee.jpg"
        />
        <img
          alt="bumblebee"
          class="resources-image"
          src="bumblebee.jpeg"
        />
        <img
          alt="leafcutter bee"
          class="resources-image"
          src="leafcutterbee.jpeg"
        />
      </div>
      <ul>
        <li>
          Plant native flowers that bloom at different stages of the season (and make sure they haven not been treated with pesticides).
        </li>
        <li>
          Provide habitat for tunnel-nesting bees, and leave brambles and bare, sunny patches in your yard for ground-nesters
        </li>
        <li>
          Purchase local and organic produce to increase the demand for bee-friendly farming practices.
        </li>
        <li>
          Go pesticide-free! Pest-control products do not discriminate, and will kill bees alongside the target pest.
        </li>
        <li>
          Get involved in local citizen science efforts to help entomologist track important data on bee populations! Check out the
           
          <a
            href="https://www.oregonbeeproject.org/"
          >
            Oregon Bee Project
          </a>
          .
        </li>
        <li>
          Spread the word and share your love of bees with others!
        </li>
      </ul>
      <button>
        <a
          href="/"
        >
          Back to Home
        </a>
      </button>
      <p
        class="credit"
      >
        All photos attributed to Emily Parker.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
